src/dbg/commandline.cpp

```diff
@@ -103,7 +103,7 @@ void CmdLineCacheLoad(JSON Root)
     EXCLUSIVE_ACQUIRE(LockCmdLine);
 
     // Clear command line
-    memset(commandLine, 0, MAX_COMMAND_LINE_SIZE);

```

Comment:
Should this really be using MAX_SETTING_SIZE instead? It might be that the bug here is the allocation of commandLine with a size of MAX_SETTING_SIZE instead of MAX_COMMAND_LINE_SIZE